src/components/Map/index.js

```diff
@@ -147,7 +148,7 @@ const Map = () => {
             geometry: `${e.lngLat.lng},${e.lngLat.lat}`,
             geometryType: "esriGeometryPoint",
             sr: 4326,
-            layers: allLayers,
+            layers: "visible:" + allLayers,
```

Comment:
Not important to change, but you could also use ES6's [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to make this string (e.g. `visible:{allLayers}`)